# mC_strand_asymmetry

Input datasets for strand asymmetry analysis are generated by pairing cytosines in the CG or CHG (CAG and CTG) context and on each strand, 
and calculating the methylation level (#C/(#C+#T)) for cytosines covered by at least 10 reads. 

Methylation at the Watson (x) and Crick (y) strand is then visualized using a scatter of points with kernel density estimations in two dimensions, 
which is calculated using the R function kde2d. 

A loess regression model is computed using the ‘loess’ function, and applied to the original data by using the R function ‘predict’.

Example dataset shows CHG methylation (CAG/CTG) on chromosome 1 of Arabidopsis sperm cells (Col-0 background).
